src/util.js

```diff
@@ -92,7 +92,9 @@ export function toGradient(number) {
 // data in format [ [A1, B1] , [A2, B2] ]
 export function downloadCsv(data, filename) {
   const fileContent = data.map((cell) => cell.join(',')).join('\n');
-  const blob = new Blob([fileContent], { type: 'text/csv' });
+  const blob = new Blob(['\ufeff', fileContent], {
+    type: 'text/csv;charset=utf-16'
```

Comment:
Curious why uft-16 rather than utf-8? We usually use utf-8, which is more backwards compatable and should be most concise when the majority of characters are ascii https://stackoverflow.com/a/496361/4651668

Reply:
I forgot that utf8 was also variable length encoding. Thought I needed 16 for the special unicode arrow symbol. Will change to utf-8, though to be honest I'm not sure that that `charset` code does anything at all. Supposedly (according to the SO links above), it's the `\ufeff` that does the trick.